detections/endpoint/windows_powershell_fakecaptcha_clipboard_execution.yml

```
@@ -0,0 +1,91 @@
+name: Windows PowerShell FakeCAPTCHA Clipboard Execution
+id: d81d4d3d-76b5-4f21-ab51-b17d5164c106
+version: 2
+date: '2025-05-14'
+author: Michael Haag, Splunk
+status: production
+type: TTP
+description: This detection identifies potential FakeCAPTCHA/ClickFix clipboard hijacking campaigns by looking for PowerShell execution with hidden window parameters and distinctive strings related to fake CAPTCHA verification. These campaigns use social engineering to trick users into pasting malicious PowerShell commands from their clipboard, typically delivering information stealers or remote access trojans.
+data_source:
+- Sysmon EventID 1
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) 
+  as lastTime FROM datamodel=Endpoint.Processes where `process_powershell`
+  AND Processes.process="*-w*h*"
+  AND (
+      (Processes.process IN ("*robot*", "*captcha*", "*CAPTCHA*", "*verify*", "*security check*", "*complete verification*"))
```

Removing the duplicate captcha string.

```suggestion
      (Processes.process IN ("*robot*", "*captcha*", "*verify*", "*security check*", "*complete verification*"))
```